js/main.js

```diff
@@ -1,9 +1,7 @@
 const table = new Table()
-
-const players = [
-  new Player('Player One', 'player'),
-  new Player('Dealer', 'dealer')
-]
+const Players = []
```

Comment:
Tā labāk nedarīt. Ar lielo burtu inicalizē klases instances mainīgos!

Reply:
Kāpēc nepatika iepriekšējais variants, kad masīvā liekam iekšā uzreiz inicializētus Player klases mainīgos?

Reply:
Te es tiešām biju sviestu dzinis, jo līdz galam nesapratu, kas šajās koda rindās ir domāts, līdz tikai nesen pieleca, ka ir jauns mainīgais const players, kur mainīgais players inicializē klasi Player ar diviem objektiem un players ir jāraksta ar mazo sākumburtu. to es sapratu tikai 09.10.2019.